File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.py
class Solution:
    def missingMultiple(self,nums,k):
        # HashSet bna lo
        mp=set()

        # Saare elements ko HashSet mei dald do
        for i in range(len(nums)):
            mp.add(nums[i])

        # k ke multiples check kartey jao
        i=1

        while True:
            x=k*i
            # Agar multiple HashSet mei nhi hai
            if x not in mp:
                return x

            i+=1

File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.java
class Solution {
    public int missingMultiple(int[] nums,int k) {
        // HashSet bna lo
        HashSet<Integer>mp=new HashSet<>();

        // Saare elements ko HashSet mei dald do
        for(int i=0;i<nums.length;i++){
            mp.add(nums[i]);
        }

        // k ke multiples check kartey jao
        for(int i=1;;i++){
            int x=k*i;
            // Agar multiple HashSet mei nhi hai
            if(!mp.contains(x)){
                return x;
            }
        }
    }
}